src/components/Page/LandingPageWidgets/RecentPagesWidget.vue

```diff
@@ -1,7 +1,16 @@
 <template>
 	<div class="recent-pages-widget">
-		<WidgetHeading :title="t('collectives', 'Recent pages')" :first="true" />
-		<div class="recent-pages-widget-container">
+		<a class="recent-pages-title"
+			@keydown.enter="toggleWidget"
+			@click="toggleWidget">
+			<WidgetHeading :title="t('collectives', 'Recent pages')" />
+			<div class="toggle-icon">
+				<ChevronDownButton :size="24"
+					:title="t('collectives', 'Expand recent pages')"
```

Comment:
Maybe adjust the title to the toggle state?

```suggestion
					:title="showRecentPages ? t('collectives', 'Collapse recent pages') : t('collectives', 'Expand recent pages')"
```

Reply:
Nice idea! I went with setting the `aria-label` for the `a` tag according to your suggestion instead - it's the better place for a label anyway, given that it's the actionable element.